File: Min and Max in Array/Min_and_Max_in_Array.cpp
class Solution {
  public:
    pair<long long, long long> getMinMax(vector<long long int> arr) {
        long long mini = 1e18,maxi =0;
        for(auto i: arr) {
            mini = min(mini,i);
            maxi = max(maxi,i);
        }
        return {mini,maxi};
    }
};

File: Min and Max in Array/Min_and_Max_in_Array.java
class Solution {
    public Pair<Long, Long> getMinMax(int[] arr) {
        long mini = Long.MAX_VALUE, maxi = Long.MIN_VALUE;
        for (long i : arr) {
            mini = Math.min(mini, i);
            maxi = Math.max(maxi, i);
        }
        return new Pair(mini, maxi);
    }
}
